docs/contributing/pull-requests/branching.md

```diff
@@ -65,31 +65,18 @@ if smaller features can be tested and released independently.
 
 ### Long-Lived Feature Branch
 
-A Long-Lived Feature Branch differs from a Short-Lived Feature Branch in three significant ways. It
-should:
+A Long-Lived Feature Branch describes a branch that serves as a respository for the changes required
```

Comment:
I think the term `repository` is slightly odd here and could be a bit confusing in git terminology.

```suggestion
A Long-Lived Feature Branch describes a branch that serves as a repository for the changes required
```

Reply:
That's a great point.  Repo is a loaded term here.  I've restructured that sentence.